databases/cassandra/cassandra.go

```diff
@@ -130,6 +139,11 @@ func (db *Database) Initialize() error {
 		}
 	}
 
+	// if it got any error at all
+	if err != nil {
+		return fmt.Errorf("Unable to initialize data base - %s ", err)
+	}
+
```

Comment:
We wouldn't want this right? Because there is an error check on line 126 which checks if it's any error other than `ErrNoKeyspace`. If it is `ErrNoKeyspace` then we created it and all is happy now.

Reply:
you are right , i saw that it is && so i added that . i'll remove it

Reply:
Cool, thanks!

Reply:
i removed it , Done